### PR TITLE
MBART supports freeze multi-lingual model when dy2sta

### PR DESCRIPTION
**Pull request by FrostML**
### PR types

Others

### PR changes

Models

### Description

MBART supports freeze multi-lingual model when dy2sta.

### CODE REVIEW

File: paddlenlp/ops/faster_transformer/transformer/faster_transformer.py [hunk 1]
```diff
@@ -1418,6 +1418,7 @@ def get_decoder(self):
 
     def forward(self,
                 input_ids=None,
+                forced_bos_token_ids=None,
```
Comment (guoshengCS) [inline paddlenlp/ops/faster_transformer/transformer/faster_transformer.py]: 看看是否和下面`forced_bos_token_id`合并

Comment (FrostML) [reply paddlenlp/ops/faster_transformer/transformer/faster_transformer.py]: Done. Thanks.

File: paddlenlp/ops/faster_transformer/transformer/faster_transformer.py [hunk 2]
```diff
@@ -1474,22 +1474,24 @@ def forward(self,
         if decoder_start_token_id is not None:
             bos_token_id = decoder_start_token_id
 
-        if forced_bos_token_id is not None:
-            if decode_strategy == "sampling":
-                trg_word = paddle.full([batch_size * num_return_sequences, 1],
-                                       forced_bos_token_id,
-                                       dtype="int32")
+        if not isinstance(forced_bos_token_id, paddle.Tensor):
```
Comment (guoshengCS) [inline paddlenlp/ops/faster_transformer/transformer/faster_transformer.py]: 动转静的时候是否是`paddle.Tensor`类型呢

Comment (FrostML) [reply paddlenlp/ops/faster_transformer/transformer/faster_transformer.py]: Done

**Review by guoshengCS — DISMISSED** [review]
LGTM

**Review by guoshengCS — APPROVED** [review]
LGTM

**Pull request merged**